src/devices/heiman.ts

```diff
@@ -1176,4 +1177,100 @@ export const definitions: DefinitionWithExtend[] = [
         },
         endpoint: (device) => ({default: 1}),
     },
+    {
+        fingerprint: [{modelID: "HS2AQ-EF-3.0", manufacturerName: "HEIMAN"}],
+        model: "HS2AQ-EF-3.0",
```

Comment:
The code looks the same as the `HS2AQ-EM`, can you merge it with that? For proper detection use `whitelabel` (search this repo for examples)